clients/osv/src/funTest/assets/vulnerabilities-by-name-and-version-expected-result.json

```diff
@@ -258,7 +258,7 @@
             },
             {
                 "type": "WEB",
-                "url": "http://jinja.pocoo.org/docs/changelog/"
+                "url": "http://jinja.pocoo.org/docs/changelog"
```

Comment:
As the trailing slash goes back and forth, should we normalize this, e.g. by always trimming it?

Reply:
can we merge this one now and I'll look into normalization into a PR on top ?

Reply:
Yes, I'll approve. Though the final newline was also again stripped. And the `GoDep` test still fails. But let's try to solve one problem at a time here.

Reply:
here is the follow up: https://github.com/oss-review-toolkit/ort/pull/8331